src/shapley_sensitivity.jl

```diff
@@ -0,0 +1,327 @@
+@doc raw"""
+
+    Shapley(dim::Int; nboot::Int=500, n_perms::Int=-1,n_var::Int=1000, n_outer::Int=100, n_inner::Int=3) 
+
+- `dim`: number of features 
+- `nboot`: for confidence interval calculation `nboot` should be specified for the number (>0) of bootstrap runs.
+- `n_perms`: The number of permutations. If None, the exact permutations method is considerd.
```

Comment:
`If None` should probably be changed to `By default it takes the value of -1 and the exact...`